packages/lib-react-components/src/Markdownz/lib/ping.js

```diff
@@ -0,0 +1,134 @@
+// This is basically a fork of https://github.com/zestedesavoir/zmarkdown/tree/master/packages/remark-ping
+// The changes made here make the plugin more flexible for supporting hashtags or other custom mentions
```

Comment:
Can you be specific about what you've changed?

Reply:
Looking through the code, if you could comment the lines that you changed, I think that would be sufficient.